src/Data/Hints.cs

```diff
@@ -383,52 +390,50 @@ public struct HeroGraveHint {
             Dictionary<string, ArchipelagoItem> SphereOnePlayer = new Dictionary<string, ArchipelagoItem>();
             Dictionary<string, ArchipelagoItem> SphereOneOthersTunic = new Dictionary<string, ArchipelagoItem>();
             Dictionary<string, ArchipelagoItem> SphereOneOthers = new Dictionary<string, ArchipelagoItem>();
-            List<string> ERSphereOneItemsAndAreas = ItemRandomizer.GetERSphereOne();
+            ItemRandomizer.PopulatePrecollected();
+            TunicPortals.VanillaPortals();
+            // StartInventoryItems is populated with your start inventory items, which are items with a location ID of -2
+            Dictionary<string, int> StartingInventory = ItemRandomizer.AddListToDict(StartInventoryItems, ItemRandomizer.PrecollectedItems);
```

Comment:
```suggestion
            Dictionary<string, int> StartingInventory = ItemRandomizer.AddListToDict(new Dictionary<string, int>(), ItemRandomizer.PrecollectedItems);
```
or change AddListToDict to make passing in a dictionary optional and create a new one if it isn't passed in

Reply:
I need to put StartInventoryItems here so I can combine the precollected items with the start inventory items from AP

Reply:
I guess, would it make more sense if I put StartInventoryItems as a global in the ArchipelagoIntegration file instead?